frontend/src/app/components/ExperimentPage.js

```diff
@@ -34,7 +35,8 @@ export default class ExperimentPage extends React.Component {
       progressButtonWidth: null,
       showDisableDialog: false,
       shouldShowTourDialog: false,
-      showTourDialog: false
+      showTourDialog: false,
+      showPreFeedbackDialog: false
```

Comment:
@chuckharmston so, should i do anything about this now, or are you just noting for the future?